engine/table/src/main/java/io/deephaven/engine/table/impl/SourceTable.java

```diff
@@ -336,6 +336,11 @@ protected void destroy() {
         if (updateSourceRegistrar != null) {
             if (locationChangePoller != null) {
                 updateSourceRegistrar.removeSource(locationChangePoller);
+                // NB: we do not want to null out any subscriptionBuffers here, as they may still be in use by a
+                // notification
+                // delivery running currently with this destroy. We also do not want to clear the table location maps as
+                // these
+                // locations may still be useful for static tables.
```

Comment:
Formatting. Spotless did you dirty. Also, it's just a copy paste, and slightly wrong.
"We do not want to null out locationChangePoller.locationBuffer here, as it may still be in use by by a..."
Drop the second sentence entirely.